package.json

```diff
@@ -56,12 +56,15 @@
 		"wrangler": "3.60.3"
 	},
 	"dependencies": {
-		"@cloudflare/privacypass-ts": "0.5.1",
+		"@cloudflare/privacypass-ts": "0.7.0-beta.0",
 		"@sentry/cli": "2.26.0",
 		"@sentry/types": "7.95.0",
 		"cron-parser": "4.9.0",
 		"promjs": "0.4.2",
 		"toucan-js": "3.3.1",
 		"typescript": "5.3.3"
+	},
+	"overrides": {
+		"@cloudflare/blindrsa-ts": "file:./vendor/cloudflare-blindrsa-ts-0.4.2.tgz"
```

Comment:
is this intentional? "Overrides are only considered in the root `package.json` file for a project" ([source](https://docs.npmjs.com/cli/v10/configuring-npm/package-json?v=true#overrides))